src/Reporter/GitLabReporter.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FriendsOfTwig\Twigcs\Reporter;
+
+use FriendsOfTwig\Twigcs\Validator\Violation;
+use JsonException;
+use Symfony\Component\Console\Output\OutputInterface;
+use function hash;
+use function implode;
+use function json_encode;
+
+final class GitLabReporter implements ReporterInterface
```

Comment:
@frankdekker 

Can we add a DocBlock here that references the corresponding documentation in GitLab? Could be useful when we encounter a bug!